lib/cli-engine/ignored-paths.js

```diff
@@ -78,7 +78,7 @@ function mergeDefaultOptions(options) {
     return Object.assign({}, DEFAULT_OPTIONS, options);
 }
 
-/* eslint-disable valid-jsdoc */
+/* eslint-disable jsdoc/check-param-names, jsdoc/require-param */
```

Comment:
A really nice thing about the separate checks is that we can disable specific checks and still enforce other things we care about!